### PR TITLE
[9.0][MIG] stock_quant_merge: port to 9.0

### PR DESCRIPTION
**Pull request by lbelorgey**

### CODE REVIEW

File: stock_quant_merge/README.rst [hunk 1]
```diff
@@ -33,7 +33,7 @@ help us smashing it by providing a detailed and welcomed `feedback
 <https://github.com/OCA/
 stock-logistics-warehouse/issues/new?body=module:%20
 stock_quant_merge%0Aversion:%20
-8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (elicoidal) [inline stock_quant_merge/README.rst:36]: remove this link following latest README template

**Review by JordiBForgeFlow — APPROVED** [review]
Functionally reviewed. OK

**Comment by JordiBForgeFlow** [comment]
:+1:

**Review by lasley — APPROVED** [review]
LGTM code, functional 👍

**Pull request merged**